docs/api.md

```diff
@@ -234,8 +234,74 @@ Result:
 ```
 
 > #### Notes
+>
+> - Clearing the local collection will mark the collection as never synchronized;
 > - Detailed API documentation for `Collection#clear()` is available [here](https://doc.esdoc.org/github.com/Kinto/kinto.js/class/src/collection.js~Collection.html#instance-method-clear).
 
+## Authentication
+
+Authenticating against a Kinto server can be achieved in [two main ways](http://kinto.readthedocs.org/en/latest/api/cliquet/authentication.html):
+
+- Using Basic Auth
+- Using an OAuth Bearer Token (eg. Firefox Account)
+
+### Using Basic Auth
+
+Simply provide an `Authorization` header option to the `Kinto` constructor:
+
+```js
+const username = "my_username";
+const password = "my_password";
+const kinto = new Kinto({
+  remote: "https://my.server.tld/v1",
+  headers: {
+    Authorization: "Basic " + btoa(`${username}:${password}`)
+  }
+});
+```
+
+Note that you can also provide this authentication header to [`#sync()`](https://doc.esdoc.org/github.com/Kinto/kinto.js/class/src/collection.js~Collection.html#instance-method-sync):
+
+```js
+kinto.collection("articles")
+  .sync({
+    headers: {Authorization: "Basic " + btoa(`${username}:${password}`)}
+  }).then(…);
+```
+
+> #### Notes
+>
+> - This authentication method should never been used in production, and passwords in clear should never be part of the source code;
```

Comment:
Also within the scope of Kinto.js authorization credentials should be read from either the user input or the URL hash but not in the source code.